tests/doc/capi/chfl_trajectory/mem_reader.c

```diff
@@ -0,0 +1,20 @@
+// Chemfiles, a modern library for chemistry file reading and writing
+// Copyright (C) Guillaume Fraux and contributors -- BSD license
+
+#include <chemfiles.h>
+#include <stdlib.h>
+#include <string.h>
+
+int main() {
+    // [example] [no-run]
```

Comment:
We can actually run this example, since it do not access any made up file like the other ones

Reply:
We run a few more examples now.